spec/requests/public_spec.rb

```diff
@@ -2,156 +2,184 @@
-
-  context 'CORS browser example' do
-    headers = {
-      'Host' => 'localhost:8080',
-      'Connection' => 'keep-alive',
-      'Cache-Control' => 'max-age=0',
-      'Origin' => 'http://localhost:8080',
-      'User-Agent' => 'Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36',
-      'Access-Control-Request-Headers' => 'accept, content-type',
-      'Accept' => '*/*',
-      'Referer' => 'http://localhost:8080/listen/234234?start=30&end=60',
-      'Accept-Encoding' => 'gzip, deflate, sdch',
-      'Accept-Language' => 'en-US,en;q=0.8',
-      'Content-Type' => 'text/plain'
-    }
-
-    it '1, has the expected CORS response' do
-      request_headers = headers.merge({ 'Access-Control-Request-Method' => 'PUT' })
-      options '/my_account/prefs', headers: request_headers
-
-      expect_empty_body
-      expect(response).to have_http_status(:ok)
-      expected_response = {
-        'Access-Control-Allow-Origin' => 'http://localhost:8080',
-        'Access-Control-Expose-Headers' => expose_headers,
-        'Access-Control-Allow-Credentials' => 'true',
-        'Access-Control-Allow-Methods' => allow_methods,
-        'Access-Control-Allow-Headers' => 'accept, content-type',
-        'Access-Control-Max-Age' => '7200'
-      }
-      expect_headers_to_include(expected_response)
-    end
-
-    it '2, has the expected CORS response' do
-      request_headers = headers.merge({ 'Access-Control-Request-Method' => 'POST' })
-      options '/my_account/prefs', headers: request_headers
-
-      expect_empty_body
-      expect(response).to have_http_status(:ok)
-      expected_response = {
-        'Access-Control-Allow-Origin' => 'http://localhost:8080',
-        'Access-Control-Expose-Headers' => expose_headers,
-        'Access-Control-Allow-Credentials' => 'true',
-        'Access-Control-Allow-Methods' => allow_methods,
-        'Access-Control-Allow-Headers' => 'accept, content-type',
-        'Access-Control-Max-Age' => '7200'
-      }
-      expect_headers_to_include(expected_response)
-    end
+def update_page(slug, text)
+  page = Comfy::Cms::Page.where(slug: slug).first
+  fragment = page.fragments.where(identifier: 'content').first
+  fragment.content = text
+  fragment.save
+end
+
+describe '/credits', type: :request do
+  create_standard_cms_pages
+
+  before do
+    update_page('credits', t = <<~MARKDOWN
+      ## If you like piña coladas
+      **And getting caught in the **rain
+      If you're not into yoga
+      If you have half a brain
+      If you like making love at midnight
+      In the dunes on the cape
+      Then I'm the love that you've looked for
+      Write to me, and escape
+    MARKDOWN
+    )
+  end
+
+  it 'should render the CMS page' do
+    get '/credits'
+
+    expect(response_body).to include(
+      <<~HTML
+        <h2 id="if-you-like-pia-coladas">If you like piña coladas</h2>
+        <p>**And getting caught in the **rain
```

Comment:
No bolding permitted?

Reply:
Ah, bad test. I'll Fix.

Bold doesn't work if goes `**word **nextword`, it needs to be `**word** nextword`.